test/integration_json_test.go

```diff
@@ -476,9 +476,9 @@ func TestRenderJsonMetadataTerraform(t *testing.T) {
 func TestRenderJsonMetadataDependencyModulePrefix(t *testing.T) {
 	t.Parallel()
 
-	tmpEnvPath := copyEnvironment(t, testFixtureRenderJsonMetadata)
+	tmpEnvPath := copyEnvironment(t, testFixtureRenderJSONMetadata)
```

Comment:
Looks like there is an inconsistency in the naming convention for test constants:
testFixtureRenderJSONMetadata - private variable
TestFixtureGetTerragruntSourceHcl - public variable